lib/axon/layers.ex

```diff
@@ -1306,4 +1306,80 @@ defmodule Axon.Layers do
     new_shape = transform(Nx.shape(x), &Axon.Shape.flatten/1)
     Nx.reshape(x, new_shape)
   end
+
+  @doc """
+  Upsamples image.
```

Comment:
This is generally used by GANs, right?

```suggestion
  Upsamples image.
  
  This can be used by GAN (Generative Adversarial Networks)
  to construct an image our of a random vector sample mimicking
  an image from a real distribution.
```